src/pages/posts/PostCreate.vue

```diff
@@ -1,83 +1,95 @@
+          <!-- EDITOR content-->
+          <q-editor toolbar-bg="grey-6" toolbar-text-color="white" toolbar-toggle-color="red-5" v-model="post.content" min-height="5rem" />
+          <!-- Form category-->
+          <q-select filled v-model="post.category" :options="options" label="Category" hint="what is the category of your post?"/>
+          <!-- Form image-->
+          <q-input v-model="post.image" filled label="Image URL" type="url" hint="what will be the image of your post?" />
+          <!-- Form Btn-->
+          <div>
+            <q-btn label="Create" type="submit" color="primary"/>
+            <q-btn label="Reset" type="reset" color="primary" flat class="q-ml-sm" />
+          </div>
+        </q-form>
+        {{response.title}}
       </div>
-    </q-form>
-   </div>
+    </div>
+    <!-- dialog de Create -->
+    <div class="q-pa-md q-gutter-sm">
+      <q-dialog v-model="persistent" persistent transition-show="scale" transition-hide="scale">
+        <q-card class="bg-green text-white" style="width: 300px">
+          <q-card-section class="row justify-center q-pt-xl q-pb-md" >
+            <q-img src="https://image.flaticon.com/icons/svg/2721/2721302.svg" style="width: 100px; height: 100px;"/>
+          </q-card-section>
+          <q-card-section class="q-pt-sm row justify-center">
+            your post was created
+          </q-card-section>
+          <q-card-actions align="right" class="bg-white text-teal">
+            <q-btn  push color="dark" label="ok" type="ok" :to="{ name: 'PostList'}"/>
+          </q-card-actions>
+        </q-card>
+      </q-dialog>
+    </div>
+    <!-- dialog de Create -->
   </div>
-</div>
 </template>
 
+<style lang="scss">
+.form__size{
+  width: 750px;
+}
+</style>
 <script>
+import handleApi from '../../helper/handleApi'
+
 export default {
   data () {
     return {
-      url: null,
-      text: null,
-      Title: null,
-      accept: false,
-      category: null,
+      post: {
+        category: '',
+        title: '',
+        description: '',
+        image: '',
+        content: ''
+      },
+      response: [],
       options: [
-        'Games', 'News', 'E-sports'
-      ]
+        'Games', 'News', 'Tecnology', 'Life'
+      ],
+      persistent: false
     }
   },
-
   methods: {
     onSubmit () {
-      this.$q.notify({
-        color: 'green-4',
-        textColor: 'white',
-        icon: 'cloud_done',
-        message: 'Submitted'
-      })
+      handleApi('createPost', {
+        payload: this.post,
+        onSuccess ({ data }) {
+          self.response = data
+          console.log(self.response)
+        },
+        onError (erro) {
+          console.log(erro)
+        }
+      }
+      )
+      this.persistent = true
     },
 
     onReset () {
-      this.Title = null
-      this.text = null
-      this.category = null
-      this.url = null
+      this.post.url = null
```

Comment:
e se fizer um for lupando o objeto e resetando os valores?

Reply:
nesse caso eu teria que colocar o objeto dentro de um array e acessar todas as keys passando o valor null com o metodo fill()?